### PR TITLE
WIP - fix MU style merging

### PR DESCRIPTION
**Pull request by patricklx**
depends on 
https://github.com/ember-cli/ember-cli/pull/8431

will rebase on it

### CODE REVIEW

**Comment by ppcano** [comment]
We have a plan to split this PR in several ones that tackle different MU issues.

File: tests/helpers/acceptance.js [hunk 1]
```diff
@@ -55,6 +57,15 @@ function createTestTargets(projectName, options) {
   options.command = options.command || 'new';
 
   return applyCommand(options.command, projectName, '--skip-npm', '--skip-bower', `--directory=${outputDir}`)
```
Comment (ppcano) [inline tests/helpers/acceptance.js]: This should be fixed by https://github.com/ember-cli/ember-cli/pull/8432. Thanks for investigating and reporting the issue.

File: tests/helpers/acceptance.js [hunk 2]
```diff
@@ -55,6 +57,15 @@ function createTestTargets(projectName, options) {
...
+    .then(() => isExperimentEnabled('MODULE_UNIFICATION') && getURLFor('canary'))
```
Comment (ppcano) [inline tests/helpers/acceptance.js]: This should be fixed by #8432. Thanks for investigating and reporting the issue.

File: blueprints/module-unification-app/files/config/environment.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ module.exports = function(environment) {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
         // e.g. 'with-controller': true
-        'ember-module-unification': true
+        EMBER_MODULE_UNIFICATION: true
```
Comment (ppcano) [inline blueprints/module-unification-app/files/config/environment.js]: This should be fixed in a different PR.

**Comment by rwjblue** [comment]
Needs a rebase, also @ppcano mind doing another review?

**Comment by ppcano** [comment]
@rwjblue Yes!, Thank you for looking at it. 

I am working with @patricklx on these MU changes. We want to merge first https://github.com/ember-cli/ember-cli/pull/8431, but we will ping you when it is finally ready.

**Comment by locks** [comment]
MU has been dropped as an initiative. Thanks everyone for your contributions!
